mesaflash.c

```diff
@@ -118,6 +119,10 @@ void print_short_usage() {
     printf("Try 'mesaflash --help' for more information\n");
 }
 
+void print_version() {
+    printf("Version: 3.2.1\n");
```

Comment:
This should use the `VERSION` macro from the start of this file.

Reply:
I totally missed that, thanks for catching it.